docs/development/add-data-guide.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[add-data-guide]]
+== Add Data guide
+
+`Add Data` in the Kibana Home application contains a collection of tutorials for setting up data flows in the Elastic stack.
+
+Each tutorial contains 3 sets of instructions
+
+1. `On Premise` Instructions for setting up a data flow when both Kibana and Elasticsearch are running on premise
```

Comment:
Change this to a bulleted list:

- **On Premise.** Set up a data flow when both Kibana and Elasticsearch are running on premise.
- **On Premise Elastic Cloud.** Set up a data flow when Kibana is running on premise and Elasticsearch is running on Elastic Cloud.
-  **Elastic Cloud.** Set up a data flow when both Kibana and Elasticsearch are running on Elastic Cloud.